File: GT-Models/src/main/java/com/DavideDalSanto/GTModels/Entities/Exercise.java
package com.DavideDalSanto.GTModels.Entities;

import lombok.*;

import javax.persistence.*;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Entity
@Table(name = "exercises")
@ToString
public class Exercise {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String name;
    private String type;
    private String muscle;
    private String equipment;
    private String difficulty;
    private String instructions;
}

File: FE/GymTonic-FE/src/app/Models/exercise.ts
export class Exercise {
  id!:number;
  name:string;
  type:string;
  muscle:string;
  equipment:string;
  difficulty:string;
  instruction:string;

  constructor(name:string, type:string, muscle:string, equipment:string, difficulty:string, instruction:string){
    this.name = name;
    this.type = type;
    this.muscle = muscle;
    this.equipment = equipment;
    this.difficulty = difficulty;
    this.instruction = instruction;
  }
}
